tests/python/test_ast_refactor.py

```diff
@@ -1039,14 +1061,23 @@ def foo() -> ti.i32:
     assert foo() == 123
 
 
-@test_utils.test()
-def test_grouped_static_for_cast():
+def _test_grouped_static_for_cast():
     @ti.kernel
     def foo() -> ti.f32:
         ret = 0.
         for I in ti.static(ti.grouped(ti.ndrange((4, 5), (3, 5), 5))):
-            tmp = I.cast(float)
+            tmp = ti.cast(I, float)
```

Comment:
```suggestion
            tmp = I.cast(float)
```
The original code should also work.